pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/WorkerUtils.java

```diff
@@ -283,15 +283,15 @@ public static PulsarClient getPulsarClient(String pulsarServiceUrl) {
     }
 
     public static PulsarClient getPulsarClient(String pulsarServiceUrl, String authPlugin, String authParams,
-                                               Boolean useTls, String tlsTrustCertsFilePath,
+                                               Boolean useTls /* Deprecated */, String tlsTrustCertsFilePath,
```

Comment:
You should remove the parameter if it is not used anymore.
Otherwise the caller may think that it is applied

Reply:
This is utils class, I'm worried about an outside project using this, so I didn't cleanup this, do we need to consider that?

Reply:
This is not a public API

It is better to remove the parameter.

The parameter means 'useTls' and if you set it you expect that you use TLS.
This is not true anymore so it can lead to subtle security bugs.

I strongly believe that we must remove this method

Reply:
Thank @eolivelli for your feedback! this has been fixed.